src/Microsoft.AspNetCore.Http.Connections.Client/Internal/WebSocketsTransport.cs

```diff
@@ -82,6 +85,8 @@ public WebSocketsTransport(HttpOptions httpOptions, ILoggerFactory loggerFactory
                 _closeTimeout = httpOptions.CloseTimeout;
             }
 
+            _webSocket.Options.SetRequestHeader(XRequestedWithName, XRequestedWithValue);
```

Comment:
Move to the top so it is possible to change the header value in `WebSocketsOptions`.

Reply:
I think we want to override this header